docs/README.md

```diff
@@ -53,9 +46,20 @@ an issue:
   * [Offscreen Rendering](tutorial/offscreen-rendering.md)
   * [Supporting macOS Dark Mode](tutorial/mojave-dark-mode-guide.md)
   * [Web embeds in Electron](tutorial/web-embeds.md)
+* [Boilerplates and CLIs](tutorial/boilerplates-and-clis.md)
+  * [Boilerplate vs CLI](tutorial/boilerplates-and-clis.md#boilerplate-vs-cli)
+  * [electron-forge](tutorial/boilerplates-and-clis.md#electron-forge)
+  * [electron-builder](tutorial/boilerplates-and-clis.md#electron-builder)
+  * [electron-react-boilerplate](tutorial/boilerplates-and-clis.md#electron-react-boilerplate)
+  * [Other Tools and Boilerplates](tutorial/boilerplates-and-clis.md#other-tools-and-boilerplates)
+
+### Advanced steps
+
+* [Application Architecture](tutorial/quick-start.md#application-architecture)
```

Comment:
It might be a bit confusing to have `Application Architecture` linked twice. Maybe the solution is to rename `Application Architecture` the first time to `Electron's Process Model` or something, and remove the link in this header?